test-e2e/tests/video_detail_mobile_test.ts

```diff
@@ -1,17 +1,21 @@
 import * as assert from "assert";
 
+import {constants, selectors} from '../utils/utils';
+
 Feature('video_detail').tag('@mobile');
 
+const agent327Title = 'Agent 327';
+const agent327Description = 'Hendrik IJzerbroot – Agent 327 – is a secret agent working for the Netherlands secret service agency. In the twenty comic books that were published since 1968, Martin Lodewijk created a rich universe with international conspiracies, hilarious characters and a healthy dose of Dutch humour.';
+
 Scenario('Video detail screen loads', ({ I }) => {
   I.amOnPage('http://localhost:8080');
-  I.scrollTo({ css: 'div[aria-label="Play Agent 327"]'});
-  I.click({ css: 'div[aria-label="Play Agent 327"]'});
+  openVideo(I, 'Agent 327');
   I.see('Agent 327');
```

Comment:
Would be nice use the `agent327Title`

Reply:
Thanks, I think that was the intention but then I never circled back to it. Fixed in latest.